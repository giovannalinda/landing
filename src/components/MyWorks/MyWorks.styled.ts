import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.section`
  padding-top: 10rem;
`

export const WorksList = styled.ul`
  display: grid;
  grid-template-columns: repeat(3, 1fr);
  grid-gap: ${theme.space[5]};
  margin-top: ${theme.space[9]};

  @media screen and (max-width: ${theme.breakpoints.mobile}) {
    grid-template-columns: 1fr;
  }
`
