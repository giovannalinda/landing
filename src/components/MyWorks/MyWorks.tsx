import Image from 'next/image'
import { Assets } from '~/assets'

import { Title } from '../Title'
import { WorkCard } from './WorkCard'

import * as S from './MyWorks.styled'

export function MyWorks() {
  return (
    <S.Container id='works'>
      <Title>My Selected Work</Title>

      <S.WorksList>
        <li>
          <WorkCard title='Eduguard - E-learning Website'>
            <Image src={Assets.ImgWork1} width='424' height='332' />
          </WorkCard>
        </li>

        <li>
          <WorkCard title='Eduguard - E-learning Website'>
            <Image src={Assets.ImgWork2} width='424' height='332' />
          </WorkCard>
        </li>

        <li>
          <WorkCard title='Huma Marketing Agency'>
            <Image src={Assets.ImgWork3} width='424' height='332' />
          </WorkCard>
        </li>

        <li>
          <WorkCard title='Beeryblog Blog Website'>
            <Image src={Assets.ImgWork4} width='424' height='332' />
          </WorkCard>
        </li>

        <li>
          <WorkCard title='Educare Elearning website'>
            <Image src={Assets.ImgWork5} width='424' height='332' />
          </WorkCard>
        </li>

        <li>
          <WorkCard title='Blogy Blog Website'>
            <Image src={Assets.ImgWork6} width='424' height='332' />
          </WorkCard>
        </li>
      </S.WorksList>
    </S.Container>
  )
}
