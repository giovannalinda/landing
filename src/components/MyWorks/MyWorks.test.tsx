import { render } from '@testing-library/react'

import { MyWorks } from './MyWorks'

describe('MyWorks', () => {
  it('should match snapshot', () => {
    const { container } = render(<MyWorks />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
