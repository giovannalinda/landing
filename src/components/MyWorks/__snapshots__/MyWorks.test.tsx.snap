// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyWorks should match snapshot 1`] = `
<section
  class="sc-hTtwUo gTUKPF"
  id="works"
>
  <h1
    class="sc-hAZoDl cKtAR"
  >
    My Selected Work
  </h1>
  <hr
    class="sc-fnykZs hxMeGu"
  />
  <ul
    class="sc-jgbSNz kdZbGu"
  >
    <li>
      <button
        aria-label="Eduguard - E-learning Website"
        class="sc-BeQoi ipgzPe"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27424%27%20height=%27332%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <div
          class="sc-kgUAyh RRsqt"
        >
          <h3>
            Eduguard - E-learning Website
          </h3>
          <div
            class="sc-gsnTZi freJdh"
          >
            <span>
              VIEW PROJECT
            </span>
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.0677 11.9929L18.818 7.75739L17.4061 9.17398L19.2415 11.0032L0.932469 11.0012L0.932251 13.0012L19.2369 13.0032L17.4155 14.8308L18.8321 16.2426L23.0677 11.9929Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </button>
    </li>
    <li>
      <button
        aria-label="Eduguard - E-learning Website"
        class="sc-BeQoi ipgzPe"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27424%27%20height=%27332%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <div
          class="sc-kgUAyh RRsqt"
        >
          <h3>
            Eduguard - E-learning Website
          </h3>
          <div
            class="sc-gsnTZi freJdh"
          >
            <span>
              VIEW PROJECT
            </span>
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.0677 11.9929L18.818 7.75739L17.4061 9.17398L19.2415 11.0032L0.932469 11.0012L0.932251 13.0012L19.2369 13.0032L17.4155 14.8308L18.8321 16.2426L23.0677 11.9929Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </button>
    </li>
    <li>
      <button
        aria-label="Huma Marketing Agency"
        class="sc-BeQoi ipgzPe"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27424%27%20height=%27332%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <div
          class="sc-kgUAyh RRsqt"
        >
          <h3>
            Huma Marketing Agency
          </h3>
          <div
            class="sc-gsnTZi freJdh"
          >
            <span>
              VIEW PROJECT
            </span>
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.0677 11.9929L18.818 7.75739L17.4061 9.17398L19.2415 11.0032L0.932469 11.0012L0.932251 13.0012L19.2369 13.0032L17.4155 14.8308L18.8321 16.2426L23.0677 11.9929Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </button>
    </li>
    <li>
      <button
        aria-label="Beeryblog Blog Website"
        class="sc-BeQoi ipgzPe"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27424%27%20height=%27332%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <div
          class="sc-kgUAyh RRsqt"
        >
          <h3>
            Beeryblog Blog Website
          </h3>
          <div
            class="sc-gsnTZi freJdh"
          >
            <span>
              VIEW PROJECT
            </span>
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.0677 11.9929L18.818 7.75739L17.4061 9.17398L19.2415 11.0032L0.932469 11.0012L0.932251 13.0012L19.2369 13.0032L17.4155 14.8308L18.8321 16.2426L23.0677 11.9929Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </button>
    </li>
    <li>
      <button
        aria-label="Educare Elearning website"
        class="sc-BeQoi ipgzPe"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27424%27%20height=%27332%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <div
          class="sc-kgUAyh RRsqt"
        >
          <h3>
            Educare Elearning website
          </h3>
          <div
            class="sc-gsnTZi freJdh"
          >
            <span>
              VIEW PROJECT
            </span>
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.0677 11.9929L18.818 7.75739L17.4061 9.17398L19.2415 11.0032L0.932469 11.0012L0.932251 13.0012L19.2369 13.0032L17.4155 14.8308L18.8321 16.2426L23.0677 11.9929Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </button>
    </li>
    <li>
      <button
        aria-label="Blogy Blog Website"
        class="sc-BeQoi ipgzPe"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27424%27%20height=%27332%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <div
          class="sc-kgUAyh RRsqt"
        >
          <h3>
            Blogy Blog Website
          </h3>
          <div
            class="sc-gsnTZi freJdh"
          >
            <span>
              VIEW PROJECT
            </span>
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M23.0677 11.9929L18.818 7.75739L17.4061 9.17398L19.2415 11.0032L0.932469 11.0012L0.932251 13.0012L19.2369 13.0032L17.4155 14.8308L18.8321 16.2426L23.0677 11.9929Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </button>
    </li>
  </ul>
</section>
`;
